Source/Godunov/iamr_godunov.cpp

```diff
@@ -57,13 +57,16 @@ Godunov::ComputeAofs ( MultiFab& aofs, const int aofs_comp, const int ncomp,
 
         if (not known_edgestate)
         {
+	    auto d_bc  = convertToDeviceVector(bcs);
+	    BCRec const* d_bc_ptr = d_bc.data();
+
```

Comment:
Does it make sense to do this once and save the results as class data?

Reply:
Darn you and your valid point.